fre/app/generate_time_averages/generate_time_averages.py

```diff
@@ -9,9 +9,22 @@ def generate_time_average(infile = None, outfile = None,
     if __debug__:
         fre_logger.info(locals()) #input argument details
     exitstatus=1
-
-    #needs a case statement. better yet, smarter way to do this? (TODO)
+        
     myavger=None
+
+    #Use cdo to merge multiple files if present
+    merged = False                            
+    if type(infile).__name__=='list' and len(infile)> 1:   #multiple files case. Generates one combined file                           
```

Comment:
This combining logic seems to be both here and in cdoTimeAverager.py, though I'm not sure this one ever is used.

Are doing the combining here once, or in each of the backend's code (cdo, frenctools, xarray) ?